crates/aptos-faucet/core/src/bypasser/auth_token.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright © Aptos Foundation
```

Comment:
You'll need to add the appropriate full Apache license header, not just the copyright line.

Reply:
Yep just did that, spoke with Josh to confirm which to do for what files.